lib/tasks/db.rake

```diff
@@ -0,0 +1,16 @@
+namespace :db do
+  desc "Add constraint to votes on topics. Vote >= 0"
+  task add_constraints: :environment do
+    ActiveRecord::Base.connection.execute <<-SQL
+      ALTER TABLE topics ADD CONSTRAINT positive_votes CHECK (votes >= 0);
```

Comment:
:cake: Indeed, validations in postgresql are awesome :)

I tested it, now and it seems like rails is rolling back the migration and then throws a 500 error in the background. So not ideal... Is there a way to tell rails how to handle a error of this kind differently?